---
title: Anchor Clustering for Large-scale Immune Repertoire Sequencing Data
event_url: https://www.uoguelph.ca/bioinformatics/news/2023/01/winter-2023-binf-virtual-seminar-series

location: Summerlee Science Complex Room 1511
address:
  street: 50 Stone Road East Guelph
  city: Guelph
  region: Ontario
  country: Canada

summary: Anchor Clustering was applied to characterize immune repertoires in dogs with neurologic disease across two different anatomical compartments. 

abstract: Adaptive immunity is mediated by antigen receptors on lymphocytes. These receptors are non-germline encoded and are created at random in every lymphocyte. Lymphocyte antigen receptors facilitate the recognition of foreign or self-antigen, and the abundance of specific clones that are encountered in an individual reflects prior contact. Consequently, understanding the composition of an individualâ€™s immune repertoire, i.e. the range of lymphocyte antigen receptor sequences, can yield insights into infectious or immune-mediated conditions. Given the vastness of the antigen receptor sequence space, it is essential to reduce its complexity by identifying similar, functionally related sequences by clustering. However, the clustering of immune repertoire data is challenging due to the computational cost associated with pairwise sequence comparison. To overcome this limitation, I developed Anchor Clustering, an unsupervised clustering method designed to identify similar sequences from millions of antigen receptor gene sequences was created. The results demonstrate that Anchor Clustering is faster than existing pairwise comparison clustering methods while achieving similar clustering quality.