---
# Display name
title: Haiyang Chang

# Name pronunciation (optional)
# name_pronunciation: Haiyang Chang

# Full name (for SEO)
first_name: Haiyang
last_name: Chang

# Status emoji
status:
  icon: ðŸ˜‰

# Is this the primary user of the site?
superuser: true

# Highlight the author in author lists? (true/false)
highlight_name: true

# Role/position/tagline
role: Bioinformatician/Computational Biologist/Data Scientist

# Organizations/Affiliations to display in Biography blox
#organizations:
#  - name: OpenAI
#    url: https://openai.com/

# Social network links
# Need to use another icon? Simply download the SVG icon to your `assets/media/icons/` folder.
profiles:
  - icon: brands/github
    url: https://github.com/skylerchang
  - icon: brands/linkedin
    url: https://www.linkedin.com/in/haiyang-skyler-chang/
  - icon: academicons/google-scholar
    url: https://scholar.google.ca/citations?user=xCn3JkAAAAAJ&hl=en



interests:
  - AI models in multi-omics
  - Drug discovery and design 
  - Cancer Genomics 

education:
  - area: PhD Bioinformatics
    institution: University of Guelph
    date_start: 2016-01-01
    date_end: 2020-12-31
    summary: |
      - First-author paper at 1 BMC Bioinformatics
      - Co-author papers at 2 Frontiers in Immunology and 1 in BMC Veterinary Research
      - Co-author paper at 1 IEEE conferences 

      Research experience:
      - Developed a Python-based antigen receptor Clustering Tool, processing one million unique sequences in 5 minutes on a MacBook Air with an M2 chip, maintaining high clustering quality comparable to established tools.
      - Designed and implemented an R Shiny dashboard  for clonotype clustering visualization, incorporating interactive network graphs, phylogenetic trees, and logo representation plots, enhancing the exploration of high-publicity clusters as potential immune signatures.
      - Collaborated on a pioneering project to conduct T cell receptor (TCR) repertoire analysis using single-cell RNA sequencing in both healthy cats and dogs and  perform the TCR clustering analysis.
      - Executed comprehensive comparative analysis of immune repertoires in dog neurological diseases across two anatomical compartments, identifying novel ante-mortem diagnostic tools through clonotype clustering and sequence pattern similarity analysis.

    
    button:
      text: 'Read Thesis'
      url: 'https://example.com'
  - area: MSc Bioinformatics
    institution: University of Guelph
    date_start: 2016-01-01
    date_end: 2020-12-31
    summary: |
      GPA: 3.95/4.0

      Major Courses:
      - Bioinformatics Programming 
      - Machine Learning
      - Genomic Methods 
      - Statistics in Bioinformatics
      - Bioinformatics Software tools
    button:
      text: 'Read Thesis'
      url: 'https://atrium.lib.uoguelph.ca/items/87dd161d-56e6-4b35-a878-05671a4f005b'
  - area: MSc Biology
    institution: Dalian University of Technology
    date_start: 2016-01-01
    date_end: 2020-12-31
    summary: |
      GPA: 3.8/4.0

      Research experience:
      - Conducted experiments involving the construction and expression of plasmid vectors of human glutamate oxaloacetate transaminases, performing protein purification and enzymatic activity assessment.
      - Optimized protein expression conditions, successfully published the crystallization structures in the Protein Data Bank (PDB) with IDs: 3WZF and 5AX8.
      - Co-author papers at Protein Expression and Purification and BioScience Trends.
work:
  - position: Graduate and Undergraduate Teaching Assistant
    company_name: University of Guelph
    company_url: ''
    company_logo: ''
    date_start: 2019-09-01
    date_end: 2023-04-30
    summary: |
      Responsibilities include:
      - Prepared educational materials and hands-on labs for lecture sessions on Unix commands, high-performance computing (HPC), and bioinformatics sequencing analysis (genome assembly, bulk RNA-seq, SNP anlysis, ATAC-seq and ChIP-seq.)
      - Led weekly lab sessions and tutorials for undergraduate students in statistics and R software, providing support for statistical modeling, data analysis, and visualization.
  - position: Research Assistant
    company_name: University of Guelph
    company_url: ''
    company_logo: ''
    date_start: 2018-09-01
    date_end: 2018-12-01
    summary: |
      Responsibilities include:
      - Created a shell pipeline script leveraging an evolutionary algorithm to identify high-similarity antigen receptors, streamlining the analysis process from Illumina sequencing platform to HPC platform.
      - Conducted immune sequencing analysis for three canine disease projects (lymphoma, minimal residual disease, and inflammatory conditions), contributing to one co-authored academic paper.

# Skills
# Add your own SVG icons to `assets/media/icons/`
skills:
  - name: Technical Skills
    items:
      - name: Python
        description: ''
        percent: 80
        icon: code-bracket
      - name: Data Science
        description: ''
        percent: 100
        icon: chart-bar
      - name: SQL
        description: ''
        percent: 40
        icon: circle-stack
  - name: Hobbies
    color: '#eeac02'
    color_border: '#f0bf23'
    items:
      - name: Hiking
        description: ''
        percent: 60
        icon: person-simple-walk
      - name: Cats
        description: ''
        percent: 100
        icon: cat
      - name: Photography
        description: ''
        percent: 80
        icon: camera

languages:
  - name: English
    percent: 100
  - name: Chinese
    percent: 75
  - name: Portuguese
    percent: 25

# Awards.
#   Add/remove as many awards below as you like.
#   Only `title`, `awarder`, and `date` are required.
#   Begin multi-line `summary` with YAML's `|` or `|2-` multi-line prefix and indent 2 spaces below.
awards:
  - title: Neural Networks and Deep Learning
    url: https://www.coursera.org/learn/neural-networks-deep-learning
    date: '2023-11-25'
    awarder: Coursera
    icon: coursera
    summary: |
      I studied the foundational concept of neural networks and deep learning. By the end, I was familiar with the significant technological trends driving the rise of deep learning; build, train, and apply fully connected deep neural networks; implement efficient (vectorized) neural networks; identify key parameters in a neural networkâ€™s architecture; and apply deep learning to your own applications.
  - title: Blockchain Fundamentals
    url: https://www.edx.org/professional-certificate/uc-berkeleyx-blockchain-fundamentals
    date: '2023-07-01'
    awarder: edX
    icon: edx
    summary: |
      Learned:
      - Synthesize your own blockchain solutions
      - Gain an in-depth understanding of the specific mechanics of Bitcoin
      - Understand Bitcoinâ€™s real-life applications and learn how to attack and destroy Bitcoin, Ethereum, smart contracts and Dapps, and alternatives to Bitcoinâ€™s Proof-of-Work consensus algorithm
  - title: 'Object-Oriented Programming in R'
    url: https://www.datacamp.com/courses/object-oriented-programming-with-s3-and-r6-in-r
    certificate_url: https://www.datacamp.com
    date: '2023-01-21'
    awarder: datacamp
    icon: datacamp
    summary: |
      Object-oriented programming (OOP) lets you specify relationships between functions and the objects that they can act on, helping you manage complexity in your code. This is an intermediate level course, providing an introduction to OOP, using the S3 and R6 systems. S3 is a great day-to-day R programming tool that simplifies some of the functions that you write. R6 is especially useful for industry-specific analyses, working with web APIs, and building GUIs.
---

## About Me

ðŸ‘‹ Hi, Iâ€™m Haiyang, but you can call me Skyler. I am a dedicated bioinformatician with a robust background in immunology and a deep passion for data analytics. My research primarily focuses on clustering and characterizing immune repertoires in companion animals using machine learning. I bring a unique blend of biological expertise and analytical skills to the table, proficient in advanced techniques such as next-generation sequencing, single-cell RNA sequencing, RNA sequencing, ATAC sequencing, and ChIP sequencing analysis. I extensively use Python and R to mine genomic data, employing both machine learning and deep learning to innovate. My goal is to drive forward innovations in genomic research and healthcare, contributing to transformative advances in the field.